app/build.gradle

```diff
@@ -58,4 +58,11 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
     androidTestImplementation "androidx.compose.ui:ui-test-junit4:$compose_version"
     debugImplementation "androidx.compose.ui:ui-tooling:$compose_version"
+
+
+    implementation 'com.squareup.retrofit2:retrofit:2.8.2'
```

Comment:
Added Retrofit Dependencies in order to allow for an online dataset to be imported into the program